routes/routes_test.go

```diff
@@ -146,3 +146,45 @@ func testRootV3Route(t *testing.T, when spec.G, it spec.S) {
 		})
 	})
 }
+
+func testAppsGetRoute(t *testing.T, when spec.G, it spec.S) {
```

Comment:
this test is never invoked. need to register within TestRouter()

Reply:
Nice catch, thanks!
https://github.com/cloudfoundry/cf-k8s-api/pull/30/commits/f34aa7d0108ccd045a77d8cce2610b79322de050